solutions/joi-19-cake-circle.mdx

```diff
@@ -0,0 +1,180 @@
+---
+id: joi-19-cake-circle
+source: JOI 2019
+title: Cake Circle
+author: Dustin Miao
+---
+
+# Explanation
+
+First, given some set of cakes $S$, it is always optimal to arrange them in sorted order by color. To prove this, consider inductively adding the cakes in ascending order. It can be shown that placing the cake between the greatest one added so far and the smallest one is always optimal. Thus, total color is $2(\max(S) - \min(S))$. 
```

Comment:
> It can be shown

doesn't really explain why this is the case

Reply:
It's just a proof sketch

Reply:
why not?

Reply:
bc its intuitive and the math doesnt really clarify anything

Reply:
fine, whatever
```suggestion
First, given some set of cakes $S$, it is always optimal to arrange them in sorted order by color. To prove this, consider inductively adding the cakes in ascending order. It can be shown that placing the cake between the greatest one added so far and the smallest one is always optimal. Thus, the total color is $2(\max(S) - \min(S))$. 
```